### DISCUSSION TITLE
Testing rel. 202305xx

### DISCUSSION THREAD

**Discussion by HJLK** [rern/rAudio#1323] [Testing]
I used the image **rAudio-64bit-20230521.img** on my Pi3B+, from scratch.

1) restoring from **/boot/backup.gz** (from my last UPDATE-version) did not work: no wlan, so I dropped backup.gz, and made /boot/wifi. That worked OK, and I manually restored all settings I use: LCDChar, Power Button, Rotary Encoder (worked OK).

2) Power button (on pin 5/7) does not do anything. No shutdown from 7, nor startup from 5.

3) My LCDChar 20x4 did not work: when manually started **lcdchar.py**  it gave the error that module **RPLCD** could not be found. I had to install that via **$ pip install RPLCD**. That did the trick.

4) I used my startup/shutdown.sh in /boot to light LEDs via Python. This did not work anymore: module RPi.GPIO could not be found. In previous images it was default present. When I tried to install it via _pip_, I got an error: building the "wheel" failed:
 "error: command 'gcc' failed: No such file or directory". 
So the gcc-compiler is not present. so it seems. How can I install this?

5) Song lyrics: what is changed since the last image? I cannot choose lyrics (only bio and add similar). When I activate lyrics, I have to insert an internet address... And as I have no lyrics in my mp3's, the recommendation is to deactivate this? 

So far for the moment, hoping for a few reactions...

=============================================
Version is now **20230528**.

ad 2): Power buttons off=7 and on=5 are working OK (as promised and expected) now.

ad 4): I downloaded **gcc-12.1.0-2.1-aarch64.pkg.tar.xz** from **https://archlinuxarm.org/packages/aarch64/gcc** and installed via pacman:

**$ pacman –U gcc-12.1.0-2.1-aarch64.pkg.tar.xz**

That went OK, and after that I could **$ pip install RPi.GPIO** without problems. And my scripts in /boot worked as expected.

**Comment by Ronaldi111097** [comment]
Boleh di coba
